docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -520,3 +546,667 @@ csharp_style_throw_expression = true:suggestion
+| False | Leave statements and member declarations on different lines | 
+
+#### Applied
+```csharp
+//csharp_preserve_single_line_statements = true
+int i = 0; string name = "John";
+```
+
+```csharp
+//csharp_preserve_single_line_statements = false
+int i = 0; 
+string name = "John";
+```
+
+#### Example editorconfig file:
+```
+# CSharp formatting settings:
+[*.cs]
+csharp_preserve_single_line_statements = true
+``` 
+
+### <a name="wrapping_block">Leave Block on Single Line</a>
+|  Option Name | `csharp_preserve_single_line_blocks` |
+| ------------- |:-------------:|
+| **Applicable Languages** | C#
+
+| Value | Description |
+| ------------- |:-------------|
+| True | Leave block on single line | 
+| False | Leave block on separate lines | 
+
+#### Applied
+```csharp
+//csharp_preserve_single_line_blocks = true
+public int Foo { get; set; }
+```
+
+```csharp
+//csharp_preserve_single_line_blocks = false
+public int MyProperty
+{ 
+    get; set;
+}
+```
+
+#### Example editorconfig file:
+```
+# CSharp formatting settings:
+[*.cs]
+csharp_preserve_single_line_blocks = true
+``` 
+
+# <a name="naming"> Naming Conventions </a>
+## Overview
+**Rule Format:**<br>
+namingRuleTitle:<br>
+`dotnet_naming_rule.<namingRuleTitle>.symbols = <symbolTitle>`<br>
+`dotnet_naming_rule.<namingRuleTitle>.style = <styleTitle>`<br>
+`dotnet_naming_rule.<namingRuleTitle>.severity = none|suggestion|warning|error`<br>
+<br>
+symbolTitle:<br>
+`dotnet_naming_symbols.<symbolTitle>.applicable_kinds = class, struct, interface, enum, property, method, field, event, namespace, delegate, type_parameter`<br>
+`dotnet_naming_symbols.<symbolTitle>.applicable_accessibilities = public, internal, private, protected, protected_internal`<br>
+`dotnet_naming_symbols.<symbolTitle>.required_modifiers = abstract, async, const, readonly, static`<br>
+<br>
+styleTitle:<br>
+`dotnet_naming_style.<styleTitle>.capitalization = pascal_case|camel_case|first_word_upper|all_upper|all_lower`<br>
+`dotnet_naming_style.<styleTitle>.required_prefix = string`<br>
+`dotnet_naming_style.<styleTitle>.required_suffix = string`<br>
+`dotnet_naming_style.<styleTitle>.word_separator = string`<br>
+
+## Writing a Naming Convention
+For naming conventions you must specify **symbols**, **style**, and a **severity**. Naming conventions should be ordered from most-specific to least-specific. The first rule encountered that can be applied, will be the only rule applied. 
+
+### Severity
+The following are valid options for the severity of a naming style rule
+ `none`, `silent`, `suggestion`, `warning`, `error`.
+
+ `none` and `silent` are synonymous and mean that no indication of any kind should be shown to the user. This has the effect of disabling this rule.
+
+ `suggestion` means that the user will be shown the following in the errorlist: and the following in the IDE. The `suggetion` severity will allow the naming rule to run, but it will not cause the build to break.

```

Comment:
I don't understand what this is saying.

Also, should it be "Error List", not "errorlist"